src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -39,12 +40,14 @@ export class RundownTimelineService implements RundownService {
   public async activateRundown(rundownId: string): Promise<void> {
     await this.assertNoRundownIsActive()
     const rundown: Rundown = await this.rundownRepository.getRundown(rundownId)
-
+    const snapBefore = rundown.getInfinitePiecesMap()
     rundown.activate()
 
     await this.buildAndPersistTimeline(rundown)
-
-    this.rundownEventEmitter.emitInfinitePiecesUpdatedEvent(rundown)
+    const snapAfter = rundown.getInfinitePiecesMap()
+    if (isDeepStrictEqual(snapBefore, snapAfter)) {
```

Comment:
Am i mistaken or do we not want it to be:
`if (!isDeepStrictEqual(snapBefore, snapAfter)` ?

Reply:
I believe order makes no diff